annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/tag/BodyTypesTest.java

```diff
@@ -0,0 +1,354 @@
+package eu.europeana.annotation.client.integration.webanno.tag;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.stanbol.commons.exception.JsonParseException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import eu.europeana.annotation.client.AnnotationSearchApiImpl;
+import eu.europeana.annotation.definitions.model.Annotation;
+import eu.europeana.annotation.definitions.model.body.GraphBody;
+import eu.europeana.annotation.definitions.model.body.TagBody;
+import eu.europeana.annotation.definitions.model.body.impl.EdmPlaceBody;
+import eu.europeana.annotation.definitions.model.body.impl.FullTextResourceBody;
+import eu.europeana.annotation.definitions.model.body.impl.SemanticTagBody;
+import eu.europeana.annotation.definitions.model.body.impl.TextBody;
+import eu.europeana.annotation.definitions.model.entity.impl.EdmPlace;
+import eu.europeana.annotation.definitions.model.graph.Graph;
+import eu.europeana.annotation.definitions.model.search.SearchProfiles;
+import eu.europeana.annotation.definitions.model.search.result.AnnotationPage;
+import eu.europeana.annotation.definitions.model.target.Target;
+import eu.europeana.annotation.definitions.model.vocabulary.BodyInternalTypes;
+import eu.europeana.annotation.definitions.model.vocabulary.MotivationTypes;
+
+public class BodyTypesTest extends BaseTaggingTest {
```

Comment:
This class is used for search tests, please rename it accordingly

Reply:
Class renamed to SearchTypesTest